client/tests/e2e/snapshots/saved/_candidate-test-plan_24_1.html

```diff
@@ -697,7 +697,7 @@ <h1 class="disclosure-heading">
                                       >esmeralda-baggins</a
                                     ></b
                                   >
-                                  on April 01, 2025.
+                                  on April 08, 2025.
```

Comment:
Realized this was accidentally reintroduced as part of the snapshots with the css re-work. https://github.com/w3c/aria-at-app/pull/1361/commits/a637884800157e9d4aeb9b0deba6d6264a2ac226 in #1361 should fix it. But let me know if you'd like me to just pull that into it's own PR instead

Reply:
Fine with it being fixed over there. Thanks!